### PR TITLE
refactor(atoms): SfBreadcrumbs

### PR DESCRIPTION
**Pull request by justyna-13**
# Scope of work
Improved breadcrumbs

# Screenshots of visual changes

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed

If applicable:

- [x] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/shared/styles/components/atoms/SfBreadcrumbs.scss [hunk 1]
```diff
@@ -1,37 +1,42 @@
...
-      1.6,
-      var(--body-font-family-secondary)
-    )
+    var(--font-weight-regular),
+    var(--font-size-small),
+    1.6,
+    var(--body-font-family-secondary)
   );
   &__list {
+    padding: var(--breadcrumbs-list, var(--spacer-small));
```
Comment (martaradziszewska) [inline packages/shared/styles/components/atoms/SfBreadcrumbs.scss]: we shouldn't change padding here, try to not use any outer positioning for components (like outer margins). The way they're positioned in the layout should be determined in the outer environment. :)

File: packages/shared/styles/components/atoms/SfBreadcrumbs.scss [hunk 2]
```diff
@@ -1,37 +1,42 @@
...
     display: flex;
-    padding: 0;
-    margin: 0;
+    flex-wrap: wrap;
     list-style: none;
   }
   &__item {
+    &:first-child {
+      padding: var(
```
Comment (martaradziszewska) [inline packages/shared/styles/components/atoms/SfBreadcrumbs.scss]: I think first-child shouldn't have padding on the left side :)

File: packages/shared/styles/components/atoms/SfBreadcrumbs.scss (hunk 1 above)
Comment (aniamusial) [reply packages/shared/styles/components/atoms/SfBreadcrumbs.scss]: I agree with Marta, also remember to add the property name to var, in this case it should be "--breadcrumbs-list-padding"

File: packages/shared/styles/components/atoms/SfBreadcrumbs.scss [hunk 3]
```diff
@@ -1,37 +1,42 @@
...
-    color: var(--breadcrumbs-link, var(--c-text));
+    color: var(--breadcrumbs-link, var(--c-link));
     cursor: pointer;
+    &:hover {
+      color: var(--breadcrumbs-hover, var(--c-link-hover));
+    }
   }
   &__current {
-    color: var(--breadcrumbs-current, var(--c-text-muted));
+    color: var(--breadcrumbs-current, var(--c-primary-variant));
```
Comment (aniamusial) [inline packages/shared/styles/components/atoms/SfBreadcrumbs.scss]: It should be simply `--c-primary` :) Variants are used for actions like :hover, :active for buttons etc.

File: packages/shared/styles/components/atoms/SfBreadcrumbs.scss (hunk 1 above)
Comment (justyna-13) [reply packages/shared/styles/components/atoms/SfBreadcrumbs.scss]: deleted :)

File: packages/shared/styles/components/atoms/SfBreadcrumbs.scss (hunk 2 above)
Comment (justyna-13) [reply packages/shared/styles/components/atoms/SfBreadcrumbs.scss]: deleted :)

File: packages/shared/styles/components/atoms/SfBreadcrumbs.scss (hunk 3 above)
Comment (justyna-13) [reply packages/shared/styles/components/atoms/SfBreadcrumbs.scss]: done :)

File: packages/shared/styles/components/atoms/SfBreadcrumbs.scss [hunk 4]
```diff
@@ -1,37 +1,38 @@
...
       &::before {
-        padding: var(--breadcrumbs-item-before-padding, 0 var(--spacer) 0 0);
+        padding: var(--breadcrumbs-item-before-padding, 0 var(--spacer-small));
         content: var(--breadcrumbs-item-seperator, "|");
       }
     }
   }
   &__link {
-    color: var(--breadcrumbs-link, var(--c-text));
+    color: var(--breadcrumbs-link, var(--c-link));
```
Comment (pspaczek) [inline packages/shared/styles/components/atoms/SfBreadcrumbs.scss]:
```suggestion
    color: var(--breadcrumbs-link-color, var(--c-link));
```

File: packages/shared/styles/components/atoms/SfBreadcrumbs.scss [hunk 5]
```diff
@@ -1,37 +1,38 @@
...
     cursor: pointer;
+    &:hover {
+      color: var(--breadcrumbs-hover, var(--c-link-hover));
```
Comment (pspaczek) [inline packages/shared/styles/components/atoms/SfBreadcrumbs.scss]:
```suggestion
      --breadcrumbs-link-color: var(--c-link-hover);
```

File: packages/shared/styles/components/atoms/SfBreadcrumbs.scss [hunk 6]
```diff
@@ -1,37 +1,38 @@
...
+    }
   }
   &__current {
-    color: var(--breadcrumbs-current, var(--c-text-muted));
+    color: var(--breadcrumbs-current, var(--c-primary));
```
Comment (pspaczek) [inline packages/shared/styles/components/atoms/SfBreadcrumbs.scss]:
```suggestion
    color: var(--breadcrumbs-current-color, var(--c-primary));
```

Comment (pspaczek) [reply packages/shared/styles/components/atoms/SfBreadcrumbs.scss]: why --c-primary? 
Zrzut ekranu 2020-03-5 o 14 37 01

**Pull request merged**